src/java/picard/analysis/CollectRrbsMetrics.java

```diff
@@ -63,26 +63,38 @@
         programGroup = Metrics.class
 )
 public class CollectRrbsMetrics extends CommandLineProgram {
-    static final String USAGE_SUMMARY = "Collect metrics from reduced representation bisulfite sequencing (RRBS) data.  ";
-    static final String USAGE_DETAILS = "This tool collect metrics for RRBS data, based on the methylation status of cytosine (C) " +
-            "bases in both CpG and non-CpG sites across all reads of a BAM/SAM file. For a brief primer on bisulfite sequencing and " +
-            "cytosine methylation, see the " +
-            "<a href='https://www.broadinstitute.org/gatk/guide/article?id=6330'>GATK Dictionary</a>." +
-            "<br /><br />" +
+    static final String USAGE_SUMMARY = "Collects metrics from reduced representation bisulfite sequencing (RRBS) data.  ";
+    static final String USAGE_DETAILS = "<p>This quality control tool uses reduced representation bisulfite sequencing (RRBS) data to determine" +
+            " cytosine methylation status across all reads of a BAM/SAM file.  For a brief primer on bisulfite sequencing and cytosine " +
```

Comment:
> uses reduced representation bisulfite sequencing (RRBS) data to determine cytosine methylation status across all reads of a BAM/SAM file

That's a bit of an odd way to describe what the tool does. I would say more simply that "evaluates cytosine methylation status in bisulfite sequencing (RRBS) data contained in a SAM or BAM file."